### ISSUE TITLE
Add crop option

### ISSUE THREAD

**Issue by kurtextrem** [Tapfiliate/tapatar#3]
Cropping uploaded avatars probably makes sense.

**Comment by thomask** [comment]
Definitely. I'd be happy to merge a pull request if you'd add this functionality.

**Comment by kurtextrem** [comment]
Currently thinking of embedding https://github.com/fengyuanchen/cropper into the preview div.

**Comment by thomask** [comment]
Crossed our minds here as well. Would be great if we could make this an optional feature (with optional dependency). We'd be happy to merge a pull request  :thumbsup:

**Comment by kurtextrem** [comment]
https://github.com/Tapfiliate/tapatar/pull/7 :tada:
